Test/Invoke-WhiskeyNodeTask.Tests.ps1

```diff
@@ -572,6 +592,16 @@ Describe 'Invoke-WhiskeyNodeTask.when run by build server, running Clean on alre
     cleanup
 }
 
+Describe 'Invoke-WhiskeyNodeTask.when run by build server, running initialize' {

```

Comment:
This test should be failing because the task's `SupportsInitialize` flag isn't set to `$true`. This didn't get caught because the test isn't calling `Invoke-WhiskeyTask`. 

Also, there are no assertions that the `npm install` command was run and that no other commands were run.

Reply:
Done, I checked if the node_modules folder is present. this fails now if SupportsInitialize isn't set.

Reply:
Done